### ISSUE TITLE
magisk daemon doesn't started

### ISSUE THREAD

**Issue by KernelGangster** [topjohnwu/Magisk#5167]
Device: Galaxy S21 (exynos chipset)
Android version: 12 (latest)
Magisk version name: v23
Magisk version code: 23016

Today i have tried rooting on Galaxy S21 after update latest android 12.

```
...
/dev/JsBOvA/.magisk/block/system_root on /dev/JsBOvA/.magisk/mirror/system_root type ext4 (ro,seclabel,relatime,i_version)
tmpfs on /system/bin type tmpfs (rw,seclabel,relatime)
/dev/JsBOvA/.magisk/block/system_root on /system/bin/abb type ext4 (ro,seclabel,relatime,i_version)
/dev/JsBOvA/.magisk/block/system_root on /system/bin/abx type ext4 (ro,seclabel,relatime,i_version)
...
```
Filesystem was mounted successfully.
```
127|o1s:/ $ su
No daemon is currently running!
o1s:/ $ magisk --daemon
No daemon is currently running!
```
But, magisk daemon is not started and magisk apk doesn't installed after booting system. I guess magisk got problem from new android 12 structure probably..? 

Thank you.

**Comment by yujincheng08** [comment]
plz attach bugreport

**Comment by KernelGangster** [comment]
[bugrepot.zip](https://github.com/topjohnwu/Magisk/files/7819571/bugrepot.zip)

**Comment by yujincheng08** [comment]
```
<12>[    8.326848] [7:           init:    1] init: Killed 1 additional processes from a oneshot process group for service '2rBrNxgZ3xKpn'. This is new behavior, previously child processes would not be killed in this case.
<14>[    8.327460] [7:           init:    1] init: Service 'usbd' (pid 1191) exited with status 0 oneshot service took 0.087000 seconds in background
<14>[    8.327468] [7:           init:    1] init: Sending signal 9 to service 'usbd' (pid 1191) process group...
<14>[    8.327536] [7:           init:    1] libprocessgroup: Successfully killed process cgroup uid 0 pid 1191 in 0ms
<14>[    8.327863] [7:           init:    1] init: Untracked pid 1195 received signal 9
```

Should be fixed by #5133

**Comment by vvb2060** [comment]
https://github.com/topjohnwu/Magisk/actions/workflows/build.yml?query=event%3Apush Install the latest build
